### ISSUE TITLE
Autocomplete doesn't appear with multiple cursors

### ISSUE THREAD

**Issue by jackfranklin** [VSCodeVim/Vim#3282]
**Describe the bug**
When you have multiple cursors, the autocomplete doesn't appear. If I disable the Vim plugin, it does.

If you then hit `Ctrl+Space` to bring up the list, pressing `Ctrl+n` then moves through the list _two at a time_. I suspect this is because I have 2 cursors, so each `Ctrl+n` is registered twice.

**To Reproduce**

1.  Have some code that includes `foo` twice
2.  In normal mode, on the first `foo`, hit `Cmd+D` to select it + then the next one
3.  Notice that the autocomplete doesn't appear (bug 1)
4.  Hit `Ctrl+Space` to bring it up.
5. Press `Ctrl+n`, and see that it skips through the list two at a time.

Note: I've tried this using `Cmd+D` and also `gb` to create the cursors.

**Expected behavior**
1. The autocomplete appears as it would if I wasn't in a multiple cursor situation.
2. `ctrl+n` goes through one at a time in all situations.

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 0.17.0
- VSCode version: 1.3.0
- OS: OS X High Sierra

**Comment by jackfranklin** [comment]
I did a bit more digging and found the issue with `C-n` and `C-p`, which the linked PR has a fix for.

For the first issue (the autocomplete not appearing automatically), I think it might have something to do with this code that changes how text insertions are created if you're in multi cursor mode:

```
      if (vimState.isMultiCursor) {
        vimState.recordedState.transformations.push({
          type: 'insertText',
          text: char,
          position: vimState.cursorPosition,
        });
      } else {
        vimState.recordedState.transformations.push({
          type: 'insertTextVSCode',
          text: char,
        });
      }
```

But my lack of familiarity with the code base is holding me back a bit - I'll keep trying to dig :)

**Comment by J-Fields** [comment]
Seems this was fixed by #3283, so closing it out
